### PR TITLE
add Chakra UI and customTheme

### PR DESCRIPTION
**Pull request by tomimarkus991**
closes #19

### CODE REVIEW

**Comment by Orzelius** [comment]
Let's create a habit of adding images to pr-s that change the look of the FE

File: fe/src/theme.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { extendTheme } from '@chakra-ui/react';
+
+// Let's say you want to add custom colors
```
Comment (GeorgDV) [inline fe/src/theme.ts:3]: Just a nitpick, but comments like these don't add anything

**Comment by GeorgDV** [comment]
Also im getting this error, i think its because Text components is used inside another Text component. But since the content of Home page will be redesigned anyway it i could let that slide for now.

error

**Comment by GeorgDV** [comment]
Otherwise looks good 👍.

File: fe/src/theme.ts (hunk 1 above)
Comment (Kasparsu) [reply fe/src/theme.ts:3]: I mean it adds the knowledge that you can add custom colors like this. I don't say its super valuable, but it makes it more clear imho. And its not actively hurting anybody. @tomimarkus991 your choice.

File: fe/src/App.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import { Home } from './components/pages/Home/index';
 import { About } from './components/pages/About/index';
 import { Error } from './components/pages/Error/index';
+import { Home } from './components/pages/Home/index';
```
Comment (Kasparsu) [inline fe/src/App.tsx:5]: Whats the purpose of this change? Look it in the diff.

File: fe/src/index.tsx [hunk 3]
```diff
@@ -1,12 +1,16 @@
+import { ChakraProvider } from '@chakra-ui/react';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
+import './index.css';
```
Comment (Kasparsu) [inline fe/src/index.tsx:5]: This change as well? look in the the diff.

File: fe/src/theme.ts (hunk 1 above)
Comment (GeorgDV) [reply fe/src/theme.ts:3]: Okay, i guess so. Doesn't hurt anyone.

**Review by Kasparsu — CHANGES_REQUESTED** [review]
fix weird line moves that serve no purpose. And take a look at the error @GeorgDV posted.

File: fe/src/App.tsx (hunk 2 above)
Comment (tomimarkus991) [reply fe/src/App.tsx:5]: I have this thing on that auto organizes imports and that's what happened here.

File: fe/src/index.tsx (hunk 3 above)
Comment (tomimarkus991) [reply fe/src/index.tsx:5]: I have this thing on that auto organizes imports and that's what happened here.

**Comment by tomimarkus991** [comment]
> 
> 
> Also im getting this error, i think its because Text component is used inside another Text component. But since the content of Home page will be redesigned anyway i could let that slide for now.
> 
> error

I will not change it, because we will change the frontend anyways...

**Review by Kasparsu — APPROVED** [review]

**Pull request merged**